gchart-parent/gchart/src/main/java/org/wicketstuff/gchart/ChartLibLoaderBehavior.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2017 Dieter Tremel.
+ * http://www.tremel-computer.de
+ * All rights, if not explicitly granted, reserved.
+ */
+package org.wicketstuff.gchart;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+import org.apache.wicket.Component;
+import org.apache.wicket.Page;
+import org.apache.wicket.Session;
+import org.apache.wicket.ajax.json.JSONArray;
+import org.apache.wicket.ajax.json.JSONObject;
+import org.apache.wicket.behavior.Behavior;
+import org.apache.wicket.markup.head.HeaderItem;
+import org.apache.wicket.markup.head.IHeaderResponse;
+import org.apache.wicket.markup.head.JavaScriptContentHeaderItem;
+import org.apache.wicket.markup.head.JavaScriptHeaderItem;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import static org.wicketstuff.gchart.Chart.LOADER_URL;
+
+/**
+ * Behavior to bundle the Google chart lib loading and package declaration for
+ * multiple charts on one page. All needed packages are merged in a set,
+ * so the loading is done only once as the
+ * <a href="https://developers.google.com/chart/interactive/docs/basic_multiple_charts">docs request</a>.
+ *
+ * To use add this behavior to the page (<em>not the chart</em>) and give the instance as
+ * param to every constructor of a chart component of the page.
+ * The Charts will register in the loader by {@link ChartLibLoader#addChart(org.wicketstuff.gchart.Chart)  }
+ * and lib loading is done centralized.
+ *
+ * @author Dieter Tremel
+ */
+public class ChartLibLoaderBehavior extends Behavior implements JavaScriptable, ChartLibLoader {
+
+    private static final Logger log = LoggerFactory.getLogger(ChartLibLoaderBehavior.class);
+    private static final long serialVersionUID = -3057525235753580423L;
+    public static final String LOADER_SCRIPT_ID = ChartLibLoaderBehavior.class.getSimpleName();
+
+    private Locale locale = null;
+    private boolean renderLocale = true;
+    private final List<Chart> charts = new ArrayList<>();
+    private HeaderItem headerItem = null;
+
+    @Override
+    public void bind(Component component) {
+        if (!(component instanceof Page) && component instanceof Chart) {

```

Comment:
IMO `if (!(component instanceof Page))` should be enough.
Because now it is possible to bind it to any Component that is not a Chart, but the javadoc and the error talk about Page.